easybuild/easyconfigs/p/pyBigWig/pyBigWig-0.3.18-foss-2021a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'pyBigWig'
+version = '0.3.18'
+
+github_account = 'deeptools'
+homepage = 'https://github.com/%(github_account)s/%(name)s'
+description = """A python extension, written in C, for quick access to bigBed
+ files and access to and creation of bigWig files."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
```

Comment:
@jfgrimm could you confirm this toolchain change is because of adding `SciPy-bundle`?

Reply:
Indeed

Reply:
Thanks.